tests/openssh.rs

```diff
@@ -669,11 +671,15 @@ async fn cannot_resolve() {
 
 #[tokio::test]
 async fn no_route() {
-    for err in connects_err("255.255.255.255").await {
+    let mut builder = SessionBuilder::default();
+
+    builder.connect_timeout(Duration::from_nanos(0));
```

Comment:
I _think_ a duration of 0s is actually "special" and means basically no timeout. I think you may have more luck setting this to something like 1.

Reply:
Done!